source-stripe-native/source_stripe_native/__init__.py

```diff
@@ -47,6 +48,35 @@ async def validate(
         log: Logger,
         validate: request.Validate[EndpointConfig, ResourceConfig],
     ) -> response.Validated:
+        if (
+            not validate.lastCapture
+            or (
+                validate.lastCapture.config.config.get("capture_connected_accounts")
+                == validate.config.capture_connected_accounts
+            )
+            or not validate.lastCapture.bindings
+        ):
+            resources = await all_resources(log, self, validate.config)
+            resolved = common.resolve_bindings(validate.bindings, resources)
+            return common.validated(resolved)
+
+        prev_bindings_by_name = {
+            binding.resourceConfig.name: binding.backfill
+            for binding in validate.lastCapture.bindings
+        }
+
+        for current_binding in validate.bindings:
+            resource_name = current_binding.resourceConfig.name
+            if (
+                resource_name in prev_bindings_by_name
+                and current_binding.backfill <= prev_bindings_by_name[resource_name]
+            ):
+                raise ValidationError(
+                    [
+                        "Cannot change the `capture_connected_accounts` property without backfilling all collections again."
+                    ]
+                )
+
```

Comment:
Validation logic to require a backfill of all collections when the `capture_connected_accounts` endpoint config property changes